thirdeye/thirdeye-frontend/app/pods/components/range-pill-selectors/component.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * Component to render pre-set time range selection pills and a 'custom' one using date-range-picker.
+ * @module components/range-pill-selectors
+ * @property {Object} timeRangeOptions - object containing our range options
+ * @property {Number} timePickerIncrement - determines selectable time increment in date-range-picker
+ * @property {Date} activeRangeStart - default start date for range picker
+ * @property {Date} activeRangeEnd - default end date for range picker
+ * @property {String} uiDateFormat - date format specified by parent route (often specific to metric granularity)
+ * @example
```

Comment:
NIT: missing `selectAction` in document here.

Reply:
done.